addons/default/visiosoft/profile-module/resources/lang/en/button.php

```diff
@@ -14,4 +14,5 @@
     'go_user' => 'Go to User Detail',
     'personal' => 'Personal',
     'corporate' => 'Corporate',
+    'export' => 'Export CSV',
```

Comment:
If someone wants to use the export translation as "Export" what will he name it then? Please name the translation according to the translated text. For example "export_csv"